i18n/de/shop/README.md

```diff
@@ -126,87 +122,93 @@ Wenn Sie als Kassenhändler jedoch möchten, dass die Hardware TSE an ihren eige
 
 ![Bestellung](images/order-for-operator.png "Bestellung der Übertragungen und Aktivierungen")
 
-Als Nächstes wird Ihnen die erfolgreiche Bestellung angezeigt und eine Bestätigung per Email versandt. Ihre Bestellungen können Sie jederzeit im Bereich "Shop->Bestellungen" einsehen. Die dazugehörigen Rechnungen befinden sich im Bereich "Shop->Rechnungen".
+Als Nächstes wird Ihnen die erfolgreiche Bestellung angezeigt und eine Bestätigung per Email versandt. Ihre Bestellungen können Sie jederzeit im Bereich <kbd>Shop</kbd> &rarr; <kbd>Bestellungen</kbd> einsehen. Die dazugehörigen Rechnungen befinden sich im Bereich <kbd>Shop</kbd> &rarr; <kbd>Rechnungen</kbd>.
 
-Nach der Bearbeitung der Bestellung durch das fiskaltrust System (kann 2-3 Minuten dauern) werden Ihnen die verbrauchten Ansprüche abgezogen. In unserem Beispiel sind unter "Shop &rarr; Entitlements" noch 14 von zuvor 15 übrig.
+Nach der Bearbeitung der Bestellung durch das fiskaltrust System (kann 2-3 Minuten dauern) werden Ihnen die verbrauchten Ansprüche abgezogen. In unserem Beispiel sind unter <kbd>Shop</kbd> &rarr; <kbd>Entitlements</kbd> noch 14 von zuvor 15 übrig.
 
 ![Ansprüche verbraucht](images/claims-consumed.png "Ansprüche verbraucht")
 
-Zudem können wir im Account des Kassenbetreibers unter "Firmenname &rarr; Standorte" sehen, dass für den zuvor bei der Aktivierung angegebenen Standort das Sorglospaket aktiviert wurde:
+Zudem können wir im Account des Kassenbetreibers unter <kbd>Firmenname</kbd> &rarr; <kbd>Standorte</kbd> sehen, dass für den zuvor bei der Aktivierung angegebenen Standort das Sorglospaket aktiviert wurde:
 
 ![Produkt Sorglos aktiviert](images/carefree-activated.png "Produkt Sorglos aktiviert")
  
  [comment]: <> (added with the information for new rollout Management)
 ## Rollout Management
-Mit dem Rollout Management können analog zum bisher beschriebenen Prozess auf von der fiskaltrust gmbH vorgefertigte Pläne zurück. Diese bereiten die Daten für die Produktion einer kompletten Konfiguration in einem oder mehreren Outlet auf. 
+Wie zuvor erläutert, führen Kassenhändler im Rahmen des Rollouts folgende Schritte durch:
+- Anzahl der Produkte kalkulieren
+- Einkäufe im [fiskaltrust.Portal](https://portal.fiskaltrust.de/) vorbereiten
+- Produkte erwerben
+- Produkte als Ansprüche an Kassenbetreiber übertragen
+- Ansprüche des Kassenbetreibers aktivieren  
+Mit dem neu eingeführten Rollout Management greifen Kassenhändler auf von der fiskaltrust gmbH vorgefertigte Pläne zurück. Damit werden die letzten Schritte zusammengefasst und vereinfacht. Erläuterung dazu finden Sie im [Video](https://www.youtube.com/watch?v=fzkoaGrQlYk) zum Thema. 
 
->![rolloutplan](images/process-MGMT.drawio.svg "rollout-plans-overview")
+>![rolloutplan](images/process-MGMT_DE.drawio.svg
+ "rollout-plans-overview")
 
 
-### Select a Rollout Plan
+### Auswahl eines Rolloutplans
 
 > ![Rollout Plan](images/portal_plans.png "rollout-management  &rarr; plan")
```

Comment:
pls. use red annotation and thinner, to maintain consistency with the other images

Reply:
I have learned to avoid red annotations, because a growing amount of people have problems to recognize red.
On top the orange is one of the componies colours, red is not.